pages/common/xh.md

```diff
@@ -0,0 +1,28 @@
+# xh
+
+> XH: Friendly and fast tool for sending HTTP requests.
+> More information: <https://github.com/ducaale/xh>.
+
+- Send a GET request:
+
+`xh {{httpbin.org/get}}`
+
+- Send a POST request with JSON body: `{"name": "ahmed", "age": 25}`:
+
+`xh post {{httpbin.org/post}} {{name=john}} {{age:=25}}`
+
+- Send a GET request with querystring id=5&sort=true:
```

Comment:
```suggestion
- Send a GET request with query parameters:
```

Reply:
Same as with JSON body example, I think that might be nice to visualize that.

Reply:
```suggestion
- Send a GET request with query parameters (e.g. `id=5&sort=true`):
```